terraform/modules/user_groups/main.tf

```diff
@@ -0,0 +1,213 @@
+# Developer group
+resource "aws_iam_group" "developer_group" {
+  name = "${var.region}-developer"
+}
+
+# Full access policy
+resource "aws_iam_policy" "full_access_policy" {
+  name        = "${var.region}-full-access-policy"
+  description = "Policy granting full access to AWS services in ${var.region}"
+  policy      = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": "*",
+      "Resource": "arn:aws:*:::${var.region}:*"
+    }
+  ]
+}
```

Comment:
Happy to be corrected - but doesn't this blanket allow policy render the finer-grained per-service allow policies below redundant?

Reply:
@aryanbhosale do you mind changing this? Or adjusting

Reply:
> Happy to be corrected - but doesn't this blanket allow policy render the finer-grained per-service allow policies below redundant?

Hey @devsjc , my bad, but ive removed the blanket allow policy in my [latest commit](https://github.com/openclimatefix/ocf-infrastructure/pull/496/commits/532a7588de224e771821c1a0e955a5f07ecd9203)

Reply:
> @aryanbhosale do you mind changing this? Or adjusting

I have made a new commit for this, could you please check it out and let me know if this is fine?